src/Workspaces/Core/Portable/WorkspacesResources.resx

```diff
@@ -747,6 +747,655 @@
   <data name="Refactoring_Only" xml:space="preserve">
     <value>Refactoring Only</value>
   </data>
+  <data name="regex_number_decimal_digit" xml:space="preserve">
```

Comment:
These go in as 'inline descriptions' in the completion list.  It looked better to me for them to just be lowercase like this.

Reply:
I guess you were the one that originally pushed us to our current format if I recall, hence my surprise you're violating it. :smile:

Reply:
Wait... i may be confused as to waht you're referring to.  Are you asking why "mark, spacing combining" starts with a lowercase letter?

Reply:
No, not the displayed text. The identifier name as gets generated into the source code.

Reply:
Oh.  

1. the identifier name here is so that all these can be grouped together.  Otherwise it was such a pain to find these in the list.
2. i had to abbreviate and not have the huge names because the loc tooling breaks with names that are like longer that 256 chars or something :)

Reply:
The prefix is fine and I think the names themselves are fine; convention is they just start with an uppercase character.

Reply:
I get teh complaint then.  Will fix :)

Reply:
#resolved